pvlib/temperature.py

```diff
@@ -459,6 +460,246 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
     return temp_air + temp_difference
 
 
+def faiman_dyn(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84,
+               thermal_inertia=0):
+    r'''
+    Calculate cell or module temperature using the Faiman model augmented
+    with a thermal inertia term.
+
+    The Faiman model is an empirical heat loss factor model [1]_ and is
+    adopted in the IEC 61853 standards [2]_ and [3]_.  The method of
+    accounting for thermal inertia was proposed and demonstrated
+    by Driesse in [4]_.
+
+    The model can be used to represent cell or module temperature.
+
+    Parameters
+    ----------
+    poa_global : numeric
```

Comment:
`poa_global` must have an index so it needs to be a Series, I believe.

Reply:
Correct, it must be a time series.